keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleEventType.kt

```diff
@@ -0,0 +1,6 @@
+package com.netflix.spinnaker.keel.lifecycle
+
+enum class LifecycleEventType {
+  BAKE
```

Comment:
I think because it's _only_ the enum, and not anything else, it is okay. Or at least my thought was that strong typing here gave enough benefit that it deserved an enum and not a string. Other details about how the event is monitored live in `keel-bakery`.